app/Http/Controllers/SectorController.php

```diff
@@ -57,35 +66,88 @@ public function store(Request $request)
     public function show(string $id)
     {
         $sector = Sector::findOrFail($id);
-        return view('ledger.show', [
+        return view('sectors.show', [
             'sector' => $sector
         ]);
     }
 
     /**
      * Show the form for editing the specified resource.
      */
-    public function edit(string $id)
+    public function edit(Request $request, string $id)
     {
-        $sector = Sector::findOrFail($id);
-        return view('sectors.edit', [
-            'sector' => $sector
-        ]);
+        if($request->user()->isAdmin())
```

Comment:
Refactor to use `Auth::user`.

Reply:
resolved in 59ff09f7743b739de37cce6a45b1e3a7053d17fb